frontend/components/welcome/Welcome.js

```diff
@@ -93,41 +108,60 @@ export const Welcome = () => {
         }
     }
 
-    return block_screen_with_this_text != null
-        ? html`<div class="navigating-away-banner"><h2>Loading ${block_screen_with_this_text}...</h2></div>`
-        : html`
-              <section id="title">
-                  <h1>welcome to <img src=${url_logo_big} /></h1>
-                  <!-- <a id="github" href="https://github.com/fonsp/Pluto.jl"
-                ><img src="https://cdn.jsdelivr.net/gh/ionic-team/ionicons@5.5.1/src/svg/logo-github.svg"
-            /></a> -->
-              </section>
-              <section id="mywork">
-                  <div>
-                      <${Recent}
-                          client=${client_ref.current}
-                          connected=${connected}
-                          remote_notebooks=${remote_notebooks}
-                          CustomRecent=${CustomRecent}
-                          on_start_navigation=${on_start_navigation}
-                      />
-                  </div>
-              </section>
-              <section id="open">
-                  <div>
-                      <${Open}
-                          client=${client_ref.current}
-                          connected=${connected}
-                          CustomPicker=${CustomPicker}
-                          show_samples=${show_samples}
-                          on_start_navigation=${on_start_navigation}
-                      />
-                  </div>
-              </section>
-              <section id="featured">
-                  <div>
-                      <${Featured} />
-                  </div>
-              </section>
-          `
+    /**
+     * These are the sources from which we will download the featured notebook titles and metadata.
+     * @type {import("./Featured.js").FeaturedSource[]}
+     */
+    const featured_sources = preact.useMemo(
+        () =>
+            // Option 1: configured directly
+            launch_params.featured_sources ??
+            // Option 2: configured through url and integrity strings
+            (launch_params.featured_source_url
+                ? [{ url: launch_params.featured_source_url, integrity: launch_params.featured_source_integrity }]
+                : // Option 3: default
```

Comment:
@pankgeorg You could add an option between 2 and 3 to overload the sources with those from juliahub. Something like:

```js
 // Option 1: configured directly
            launch_params.featured_sources ??
            // Option 2: configured through url and integrity strings
            (launch_params.featured_source_url
                ? [{ url: launch_params.featured_source_url, integrity: launch_params.featured_source_integrity }]
                : // Option 3: overloaded by environment
				environment.featured_sources ??
				  // Option 4: default
                  default_featured_sources.sources

Reply:
Because this PR adds the `launch_params.featured_sources` config, but for JH integration the `environment` seems better suited, right?